trade-finance-manager-ui/server/controllers/deals/index.js

```diff
@@ -1,107 +1,151 @@
 const api = require('../../api');
 const { generateHeadingText } = require('../helpers');
 const CONSTANTS = require('../../constants');
-
-const getDeals = async (req, res) => {
+const PageOutOfBoundsError = require('../../errors/page-out-of-bounds.error');
+
```

Comment:
I can't see unit tests for these changes?

Reply:
There is a unit test for PageOutOfBoundsError. If you mean the frontend pagination changes in general, these tests are in a different ticket, 6935